src/mediapipe_calculators/pytensor_ovtensor_converter.cc

```diff
@@ -90,18 +90,28 @@ class PyTensorOvTensorConverterCalculator : public CalculatorBase {
             const auto& options = cc->Options<PyTensorOvTensorConverterCalculatorOptions>();
             const auto tagOutputNameMap = options.tag_to_output_tensor_names();
             auto outputName = tagOutputNameMap.at(OVMS_PY_TENSOR_TAG_NAME).c_str();  // Existence of the key validated in GetContract
+            std::string precision = toKfsString(ovElementTypeToOvmsPrecision(inputTensor.get_element_type()));
+            if (precision == "UNDEFINED") {
+                return mediapipe::InvalidArgumentErrorBuilder(MEDIAPIPE_LOC)
+                       << "Undefined precision in input tensor: " << inputTensor.get_element_type();
+            }
+
             pythonBackend.createOvmsPyTensor(
                 outputName,
                 const_cast<void*>((const void*)inputTensor.data()),
                 shape,
-                toString(ovElementTypeToOvmsPrecision(inputTensor.get_element_type())),
+                precision,
                 inputTensor.get_byte_size(),
                 outputPyTensor,
                 true);
             cc->Outputs().Tag(OVMS_PY_TENSOR_TAG_NAME).Add(outputPyTensor.release(), cc->InputTimestamp());
         } else {
             auto& inputTensor = cc->Inputs().Tag(OVMS_PY_TENSOR_TAG_NAME).Get<PyObjectWrapper<py::object>>();
-            auto precision = ovmsPrecisionToIE2Precision(fromString(inputTensor.getProperty<std::string>("datatype")));
+            auto precision = ovmsPrecisionToIE2Precision(fromKfsString(inputTensor.getProperty<std::string>("datatype")));
```

Comment:
We should validate if inputTensor is `pyovms.Tensor` object. Otherwise it might segfault if accessed object does not have "datatype" property. You could use: PythonBackend::validateOvmsPyTensor